spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -57,7 +60,29 @@
 
   scenario "unauthenticated user fails to view a list of their adventures " \
   "associated with each of their bucket lists" do
+    user = FactoryGirl.create(:user)
+
+    bucket_list = FactoryGirl.create(
+      :bucket_list,
+      user_id: user.id,
+      is_public: false
+    )
+    adventure = FactoryGirl.create(
+      :adventure
+    )
+    bucket_list_adventure = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure`. Did you mean `bucket_list`?